content/english/python-basics/data-types/booleans.md

```diff
@@ -23,14 +23,51 @@ What are the boolean answers to these questions about you?
 | `<=`     | Less than or equal to | `>=`     | Greater than or equal to |
 | `==`     | Equal to              | `!=`     | Not equal to             |
 
+### Challenge 1
+As usual, use `print` to print out your results to the following:
 
-As usual, use <font color="blue">print</font> to print out your results to the following:
-
-<pre>
-    <font color="blue">print</font>(5 + 8 < 10)
-    <font color="blue">print</font>((3 + 5) * 6) == (65 - 17)
-</pre>
+```python
+print(5 + 8 < 10)
+print((3 + 5) * 6) == (65 - 17)
+```
 
 The first statement should return <font color="blue">False</font>. And the second should return <font color="blue">True</font>. 
 
-![alt text](../media/booleans.png "image of how booleans are used in python code") 
+![alt text](../../img/booleans.png "image of how booleans are used in python code") 
+
+### Challenge 2
+Try printing out the answers to the following expressions using `print`. If the results for any of these statements don't make sense, please ask for help! 
+
+- 54 < (10 + 32)
+- (37 / 5) != 7
+- "Hello" + "World" == "Hello World"
+- <font color="blue">False</font> == <font color="blue">False</font>
+
+### Challenge 3 
+Try making your own expressions! 
+
+### Fun Fact: Connecting Booleans with AND and OR operators
```

Comment:
To remove confusion, please don't use AND (since AND is not Python syntax) - instead use `and`.